lib/utils/string_extension.dart

```diff
@@ -146,4 +147,74 @@ extension StringCasingExtension on String {
         timeline.events.indexOf(secondEvent) >
             timeline.events.indexOf(firstEvent);
   }
+
+  bool containsWord(String word) {
+    final regExp = RegExp(
+      "\\b(?:$word)\\b",
```

Comment:
what is it? how to make it more clear for the blind of regex?

Reply:
it is easily to understand if place next to `contains` exist function
if need more clear, I think we should add docs comment for that function
![Screenshot 2023-09-12 at 11 25 55](https://github.com/linagora/twake-on-matrix/assets/12546908/cf98cf61-caa7-49a5-b4f5-a14ed0b526ef)